doc/man3/SSL_SESSION_get0_hostname.pod

```diff
@@ -15,48 +14,57 @@ SSL_SESSION_set1_alpn_selected
  const char *SSL_SESSION_get0_hostname(const SSL_SESSION *s);
  int SSL_SESSION_set1_hostname(SSL_SESSION *s, const char *hostname);
 
- void SSL_SESSION_get0_alpn_selected(const SSL_SESSION *s,
-                                     const unsigned char **alpn,
-                                     size_t *len);
- int SSL_SESSION_set1_alpn_selected(SSL_SESSION *s, const unsigned char *alpn,
-                                    size_t len);
+ void SSL_SESSION_get0_alpn_selected(const SSL_SESSION *s, const unsigned char **alpn, size_t *len);
+
+ int SSL_SESSION_set1_alpn_selected(SSL_SESSION *s, const unsigned char *alpn, size_t len);
```

Comment:
These functions were previously split to avoid long line lengths. Why make this change?